.builder/actions/aws_crt_java_build.py

```diff
@@ -0,0 +1,25 @@
+
+import Builder
+import os
+import argparse
+
+# This action is used by the musl-linux-build.sh script to build the binaries for release.
+
+
+class AWSCrtJavaBuild(Builder.Action):
+
+    def run(self, env):
+        if os.getenv("CRT_FIPS") is not None:
+            env.shell.exec("mvn", "-P", "continuous-integration", "-B", "compile",
+                           "-Dcmake.crt_fips=ON", check=True)
+        else:
+            env.shell.exec("mvn", "-P", "continuous-integration",
+                           "-B", "compile", check=True)
+
+        # allow custom python to be used
+        parser = argparse.ArgumentParser()
+        parser.add_argument('--classifier')
```

Comment:
I use the argparser instead of env_variable because, in CI, `linux-musl-armv7` needs to pass the env var into the docker, which is not easy to do.

But for "CRT_FIPS", I also want our test to get the info and verify the run time check for FIPS is working as expected.